tests/test_algorithm.py

```diff
@@ -3737,15 +3744,15 @@ def handle_data(context, data):
                 context.portfolio.cash == context.portfolio.starting_cash
             )
 
-            now = context.get_datetime()
+            today_session = normalize_date(context.get_datetime())
```

Comment:
Should we try to be explicit by using the calendar to get the label instead of normalizing?  Then we could even parameterize the test over various calendars.